helm/values.yaml

```diff
@@ -150,4 +162,32 @@ agent:
           "app_signals": { }
         }
       }
-    }
\ No newline at end of file
+    }
+  enableGpuConfig: 
+    {
+      "logs": {
+        "metrics_collected": {
+          "kubernetes": {
+            "gpu_metrics": true
```

Comment:
instead of having this as true, can we just have it an empty map in case we want to add more config options in the future?

Reply:
removed. it was a leftover from the previous revision where there were another agent daemonset specifically for GPU nodes.